eslint/babel-eslint-parser/package.json

```diff
@@ -1,32 +1,29 @@
 {
-  "name": "babel-eslint",
-  "version": "11.0.0-beta.0",
-  "description": "Custom parser for ESLint",
+  "name": "@babel/eslint-parser",
+  "version": "0.0.0",
+  "description": "ESLint parser that allows for linting of experimental syntax transformed by Babel",
   "author": "Sebastian McKenzie <sebmck@gmail.com>",
   "license": "MIT",
   "private": true,
   "repository": {
     "type": "git",
-    "url": "https://github.com/babel/babel-eslint.git"
+    "url": "https://github.com/babel/babel.git"
   },
   "bugs": {
-    "url": "https://github.com/babel/babel-eslint/issues"
-  },
-  "homepage": "https://github.com/babel/babel-eslint",
-  "scripts": {
-    "changelog": "git log `git describe --tags --abbrev=0`..HEAD --pretty=format:' * %s (%an)' | grep -v 'Merge pull request'"
+    "url": "https://github.com/babel/babel/issues"
   },
+  "homepage": "https://github.com/babel/babel",
   "engines": {
-    "node": ">=6"
+    "node": ">=10"
```

Comment:
`engines`/dep updates are in anticipation for the Babel v8 release.

Reply:
Suggested `>=10.9` to be aligned with babel 8.